### DISCUSSION TITLE
是否考虑舍弃在table.excel中配置数据表的方式，改为自动生成？

### DISCUSSION THREAD

**Discussion by jackyli86** [focus-creative-games/luban#124] [Ideas]
项目默认到处的表格，需要在 table.excel 中手动添加，是否考虑给定一个目录，默认生成一个类似table.excel中的配置，使用者只对某些需要特殊的表做特化处理，省却了手动配置的麻烦

**Comment by thw0609** [comment]
这个自己改下源码可以实现的
